tests/ext/sidecar_enabled.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Sidecar should be enabled by default on PHP 8.3
+--SKIPIF--
+<?php include 'startup_logging_skipif_unix_83.inc'; ?>
```

Comment:
The name is a bit confusing. What about `startup_logging_skip_unless_unix_83`?

Also, what about moving the files to the `includes` directory